### PR TITLE
couple more fixes after edit

### PR DESCRIPTION
**Pull request by katmurp**
Just a few things I noticed:

- tar extraction command improved as wasn't working for me
- live-preview requires a rerun to pick up changes
- one stylistic change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ $
 To view the html docs, extract them and launch a webserver, e.g. via Python, and point your browser at `http://localhost:8000`.
 
 ```zsh
-tar xf docs/workdir/target/html-docs-$(jq -r '.daml' LATEST).tar.gz
```
[the remark this answers is not in the document]
Comment (katmurp) [reply README.md:116]: Yes, it just worked for me too, it didn't earlier today curiously.

File: README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Run the following script to download the documentation tarballs from the Daml an
 
 ### Live preview
 
-Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Edits to `docs/index/` files should be immediately reflected.
+Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Rerun the script to view any edits to `docs/index/` files.
```
Comment (garyverhaegen-da) [inline README.md:86]:
```suggestion
Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Changes to files under `docs/index/` should reflect immediately, though you'll have to kill and rerun the script when adding or removing a file.
```

I'd rather fix the issue than document broken behaviour; please confirm that #65 fixes this for you.

Comment (katmurp) [reply README.md:86]: Hi, `live-preview` still requires a restart to pick up changes.

Comment (garyverhaegen-da) [reply README.md:86]: Can you confirm you still see that bug on the `fix-live-preview` branch?

Comment (katmurp) [reply README.md:86]: Still have to rerun the script to see changes.

Comment (garyverhaegen-da) [reply README.md:86]: I'm still not comfortable documenting broken behaviour.

Comment (katmurp) [reply README.md:86]: Until it's fixed, this is how it behaves.

**Review by garyverhaegen-da — APPROVED** [review]

**Pull request merged**